### PR TITLE
nicely show github action statuses that do not have a description

### PR DESCRIPTION
**Pull request by grosser**
Screen Shot 2021-03-03 at 12 03 33 PM

### CODE REVIEW

File: app/assets/javascripts/ref_status_typeahead.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ function refStatusTypeahead(options){
       var item = $("<li>");
       $ref_problem_list.append(item);
       // State and status comes from GitHub or Samson
-      item.html(status.state + ": " + status.description);
+      item.html(status.state + ": " + (status.description || status.context));
```
Comment (adammw) [inline app/assets/javascripts/ref_status_typeahead.js:76]:
```suggestion
      item.html(status.state + ": " + (status.description.presence || status.context));
```

**Review by adammw — APPROVED** [review]

File: app/assets/javascripts/ref_status_typeahead.js (hunk 1 above)
Comment (adammw) [reply app/assets/javascripts/ref_status_typeahead.js:76]: also should probably be using `item.text()` to avoid XSS

Comment (grosser) [reply app/assets/javascripts/ref_status_typeahead.js]: we have lots of users that need html:
```
Deploy.first(100).uniq { |d| d.project_id }.map { |d| CommitStatus.new(d.project, d.reference, stage: d.stage).statuses }
```
so added sanitization in the controller

Comment (grosser) [reply app/assets/javascripts/ref_status_typeahead.js]: Screen Shot 2021-03-04 at 10 41 44 AM

**Pull request merged**